### DISCUSSION TITLE
HLPE WITH DDLC MONIKA AFTER STORY

### DISCUSSION THREAD

**Discussion by idknousernameig** [Monika-After-Story/MonikaModDev#9250] [QA - tech support + general]
So I tried to download and it didn't work.

While running game code:
  File "game/0utils.rpy", line 16, in <module>
ImportError: cannot import name handlers

I try deleting the python file that have only one thing inside. NOTHING. I made sure it's in the game folder. NOTHING. I DON'T KNOW WHAT TO DO

**Comment by Wingdinggaster656** [comment]
Calm down when you describe the problem. Most people don't care how manic you are.

Have you read our wiki? It's here: https://github.com/Monika-After-Story/MonikaModDev/wiki/Installation-and-Readme
You may also need a more detailed guide, then it's another wiki page: https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#installation
Also please take a screenshot of your `/game` directory.

By the way, why did you think it would be useful to delete that Python file? Jump into a game folder and delete things you don't understand? I would be surprised if it really worked.

**Comment by idknousernameig** [comment]
Sorry about yesterday. I was really stressed. I was searching on everything and tried out things that were suggested. But lemme get it. I'm on Mac so it's kinda complicated.
<img width="577" alt="Screen Shot 2022-06-04 at 10 21 34 PM" src="https://user-images.githubusercontent.com/106859862/172036378-ac2f79e6-58ce-4a40-997e-2db9a5be851b.png">

**Comment by Wingdinggaster656** [comment]
> Sorry about yesterday. I was really stressed. I was searching on everything and tried out things that were suggested. But lemme get it. I'm on Mac so it's kinda complicated. <img alt="Screen Shot 2022-06-04 at 10 21 34 PM" width="577" src="https://user-images.githubusercontent.com/106859862/172036378-ac2f79e6-58ce-4a40-997e-2db9a5be851b.png">

Oh my, that's a completely wrong install.

Extract all things in `Monika_After_Story-0.12.8-Mod` folder into the `/game` directory.

**Comment by calvin12345678** [comment]
have you try to delete and reinstaling the game
